src/CodeGenerator/Generator/RequestSerializer/QuerySerializer.php

```diff
@@ -123,15 +150,13 @@ private function dumpArrayStructure(string $output, string $input, StructureShap
         $memberCode = implode("\n", array_map(function (StructureMember $member) use ($output) {
             $shape = $member->getShape();
             if ($member->isRequired() || $shape instanceof ListShape || $shape instanceof MapShape) {
-                $inputElement = '$input->get' . $member->getName() . '()';
                 $body = 'MEMBER_CODE';
+                $inputElement = '$input->get' . $member->getName() . '()';
             } else {
+                $body = 'if (null !== $v = INPUT_NAME) {
```

Comment:
not related to that PR, but consistent with `rest-json`

Reply:
Thank you